README.md

```diff
@@ -130,5 +130,4 @@ permalink: <some permalink>
     - [In Sinhala - සිංහලෙන්](/installation/installation_linux(sin).md) 
 - Local Setup on MacOS 
     - [In English](/installation/installation_macos.md)
-    - [In Sinhala - සිංහලෙන්](/installation/installation_macos(sin).md) 
-
+    - [In Sinhala - සිංහලෙන්](/installation/installation_macos(sin).md) 
```

Comment:
extraneous change. Please add one blank line

Reply:
I said it is a past one